dashboard/app/reporting_email.go

```diff
@@ -239,17 +245,23 @@ func emailReport(c context.Context, rep *dashapi.BugReport) error {
 	if err := json.Unmarshal(rep.Config, cfg); err != nil {
 		return fmt.Errorf("failed to unmarshal email config: %v", err)
 	}
-	to := email.MergeEmailLists([]string{cfg.Email}, rep.CC)
+	to := []string{cfg.Email}
+	cc := []string{}
 	if cfg.MailMaintainers && public {
 		to = email.MergeEmailLists(to, rep.Maintainers, cfg.DefaultMaintainers)
 	}
+	if len(to) < 1 {
+		to = email.MergeEmailLists(to, rep.CC)
```

Comment:
rep.CC contains emails from different sources.
I think we need to start by listing all sources of emails we have and think whether we want them in to/cc.
It's hard to review this w/o understanding the rules we want to achieve.